meta-mender-demo/recipes-mender/example-custom-inventory/files/mender-inventory-demo

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+KEY_NAME=Location
+KEY_VALUE=Europe
+
+echo "$KEY_NAME=$KEY_VALUE"
```

Comment:
In general I think that a better design is to only store data on the data partition, and have the script (still on the rootfs) fetch data from there. This gives the best of both worlds: The data survives, but the script can be updated.

Reply:
I'm hoping to have it fully customizable for cases where I want to quickly add 5 new inventories to an existing RO image.
But I need this only for the demo images I build to give demos, it doesn't need to end up in the reference demo image.
Will make that separation in the images.

Reply:
Removed it from the image, just exists as a helper recipe in the demo layer.

Reply:
Removed it completely to simplify the merge